$MirOS: ports/print/cups/patches/patch-backend_socket_c,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- backend/socket.c.orig	Tue Mar  7 22:07:21 2006
+++ backend/socket.c	Tue May  9 17:39:25 2006
@@ -353,6 +353,7 @@ main(int  argc,				/* I - Number of comm
 
       tbytes += nbytes;
       bufptr = buffer;
+      wbytes = 0;
 
       while (nbytes > 0)
       {
